polling-app/src/Components/Registation/registationForm.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { Form, Button, Navbar, Spinner } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { registationForm } from "../../Redux/Action/registation";
+import { connect } from "react-redux";
+
+class Registation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      password: "",
+      isLoading: "",
+      disable: false,
+      option: ""
+    };
+  }
+  handleOptionChange = e => {
+    e.preventDefault();
+    this.setState({ option: e.target.value });
+  };
+  handleUserChange = e => {
+    this.setState({ username: e.target.value });
+  };
+  handlePasswordChange = e => {
+    this.setState({ password: e.target.value });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+    this.props.registationRequest(this.state.username, this.state.password);
+    if (this.props.registerStatus && this.props.registerStatus.isRegistion) {
+      this.setState({ username: "" });
+      this.setState({ password: "" });
+    }
+  };
+  render() {
+    if (this.props.registerStatus && this.props.registerStatus.isRegistion) {
+      this.props.history.push("/login");
+    }
+    if (
```

Comment:
what is this for

Reply:
if registration is completed then push on login pages and line 40-44 have no meaning I will remove it